backend/jottacloud/jottacloud.go

```diff
@@ -0,0 +1,641 @@
+package jottacloud
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ncw/rclone/backend/jottacloud/api"
+	"github.com/ncw/rclone/fs"
+	"github.com/ncw/rclone/fs/config"
+	"github.com/ncw/rclone/fs/config/configmap"
+	"github.com/ncw/rclone/fs/config/configstruct"
+	"github.com/ncw/rclone/fs/config/obscure"
+	"github.com/ncw/rclone/fs/fserrors"
+	"github.com/ncw/rclone/fs/fshttp"
+	"github.com/ncw/rclone/fs/hash"
+	"github.com/ncw/rclone/lib/pacer"
+	"github.com/ncw/rclone/lib/rest"
+	"github.com/pkg/errors"
+)
+
+// Globals
+const (
+	minSleep          = 10 * time.Millisecond
+	maxSleep          = 2 * time.Second
+	decayConstant     = 2       // bigger for slower decay, exponential
+	defaultDevice     = "Jotta" // TODO ? : Make this a config Option?
+	defaultMountpoint = "Sync"  // TODO ? : Make this a config Option?
```

Comment:
defaultDevice usualy never changes so IDK, but defaultMountpoint could so it makes sense to make it configureable

Reply:
I agree on device. It's actually possible to create you own mountpoints but wasn't sure if how i should implement that.

Reply:
I guess if you were to implement that, it could be treated the same way as folders? Eg an mountpoint "is just" an folder, if that makes sense.